### DISCUSSION TITLE
自建服务器调用公共服务器身份码失败

### DISCUSSION THREAD

**Discussion by GyakuKaku** [xfgryujk/blivechat#178] [提问] [answered]
![2073](https://github.com/xfgryujk/blivechat/assets/30260431/87eb351e-3ac8-426c-8e05-21aabe7da2c9)
打扰了，本地启动blivechat身份码模式已经能正常使用。但我服务器上的不行。是因为我开着网页一直重试给服务器造成压力被墙了吗？如果恢复不了的我改用本地版......

**Comment by xfgryujk** [comment, marked as answer]
等DNS刷新吧

**Comment by GyakuKaku** [comment]
好的，我先用本地版的。服务器的改成1.9.0版本，应该不会再一直重试给服务器发请求了🙏

**Comment by GyakuKaku** [comment]
![8182](https://github.com/xfgryujk/blivechat/assets/30260431/9a85d1fe-3102-4bc7-b4b4-b2cf5049324b)
打扰了，这个被墙这状况在我这是可以复现的。从log上看，在没用服务器转发模式的场景下，只开一个房间每20秒调用一次心跳接口2到4个小时之后访问公共服务器就会返回500，过一段时间之后才能恢复。这个是问题或者是我配置操作导致的吗？

**Comment by xfgryujk** [comment]
> ![8182](https://private-user-images.githubusercontent.com/30260431/317574731-9a85d1fe-3102-4bc7-b4b4-b2cf5049324b.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE1OTU3NTAsIm5iZiI6MTcxMTU5NTQ1MCwicGF0aCI6Ii8zMDI2MDQzMS8zMTc1NzQ3MzEtOWE4NWQxZmUtMzEwMi00YmM3LWI0YjQtYjJjZjUwNDkzMjRiLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI4VDAzMTA1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFmZGZiZmU4NmE0ZWUwNTE2YzM0ZjQwMDFkYWFmODBmZGJjMzZjNDEyZGUwZGI5ZDlmMTg1YWRiMTllMzdlZjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HmXi9p0HWWjX3cnr20WUWqn26_CPdLXgY0h99afjIEY) 打扰了，这个被墙这状况在我这是可以复现的。从log上看，在没用服务器转发模式的场景下，只开一个房间每20秒调用一次心跳接口2到4个小时之后访问公共服务器就会返回500，过一段时间之后才能恢复。这个是问题或者是我配置操作导致的吗？

不是，这个时候正在被DDoS

**Comment by GyakuKaku** [comment]
好的明白了，感谢回复
